README.md

```diff
@@ -9,7 +9,7 @@
 - **Diffing:** reloads your UI with beautiful animations when your data changes
 - **Component-based:**  Design reusable components and share your custom UI across multiple screens of your app
 
-In our experience it makes UI-related code easier to build and maintain. Our aim is to make the UI a function of state (`UI = f(state)`), which makes `Bento` a perfect fit for Reactive Programming.
+In our experience it makes UI-related code easier to build and maintain. Our aim is to make the UI a function of state (i.e: `UI = f(state)`), which makes `Bento` a perfect fit for Reactive Programming.
```

Comment:
I disagree with the `i.e:` because it's not an example, it's just a different way of saying `Our aim is to make the UI a function of state`.

Reply:
That's what `i.e:` means: Latin `id est`, meaning `that is`, meaning "a different way of saying"

Reply:
To remember what it means I always use "eg = example given, ie = in essence" which is not correct but it fits :)

Reply:
I mistakenly read it as example ` exempli gratia`/`eg`. 👍